Sem. 03/Pract. 03/Task01.cpp

```diff
@@ -0,0 +1,197 @@
+#include <iostream>
+#include <fstream>
+
+const char* OUTPUT_FILE_NAME = "sorted.bin";
+const char* FILE_NAME = "orders.txt";
+
+const int MAX_SIZE = 25;
+
+struct Order {
+    char address[MAX_SIZE];
+    double price;
+};
+
+void saveOrder(std::ofstream& file, const Order& order) {
+    file.write((const char*) order.address, MAX_SIZE);
+    file.write((const char*) &order.price, sizeof(double));
+}
+
+void saveOrdersToFile(std::ofstream& file, const Order* orders, const int ordersCount) {
+    file.write((const char*) &ordersCount, sizeof(int));
+
+    for (int i = 0; i < ordersCount; ++i) {
```

Comment:
since we have an array of objects with fixed size we can write the whole array to the file directly